### PR TITLE
(dont merge) Scroll dev 0920

### PR DESCRIPTION
**Pull request by lispc**
merge this after GPU halo2 finsihed?

### CODE REVIEW

**Comment by lispc** [comment]
@zhenfeizhang do you have the push permission of this repo?  It seems that master has updated, and after merging there are some new problem needed to be fixed, mostly OK => Value::known..

**Comment by zhenfeizhang** [comment]
I think so. I will fix those

**Comment by lispc** [comment]
@zhenfeizhang  

I merged with scroll-dev-0818 and upgrade zkevm-circuits and fix building.   I will test this branch with our testnet data. And if you have time you can have a review.  (I run cargo clippy so the diff is large anda bit annoying... )

**Comment by zhenfeizhang** [comment]
In general LGTM. 

OTOH I noticed that this repo does not have any CI. Is this intentional?

**Comment by lispc** [comment]
> In general LGTM.
> 
> OTOH I noticed that this repo does not have any CI. Is this intentional?

should be added later https://github.com/scroll-tech/halo2-snark-aggregator/issues/27

**Comment by lispc** [comment]
common rs tests passed. So merge it now.

File: halo2-snark-aggregator-api/src/systems/halo2/expression.rs [hunk 1]
```diff
@@ -28,21 +28,9 @@ impl<A: ArithEccChip> Evaluable<A> for Expression<A::AssignedScalar> {
...
-                query_index,
-                column_index: _,
-                rotation: _,
-            } => advice(*query_index),
-            Expression::Instance {
-                query_index,
-                column_index: _,
-                rotation: _,
-            } => instance(*query_index),
+            Expression::Fixed(fixed_query) => fixed(fixed_query.column_index()),
```
Comment (lispc) [inline halo2-snark-aggregator-api/src/systems/halo2/expression.rs:31]: it should be fixed_query.index rather than column_index

**Pull request merged**